frontend/test/metabase-lib/lib/Dimension.unit.spec.js

```diff
@@ -801,6 +801,62 @@ describe("Dimension", () => {
           expect(base_type).toBe("type/Integer");
         });
 
+        it.each([
+          {
+            field: ["field", PRODUCTS.CATEGORY.id, null],
```

Comment:
use an object-styled parameter in the table so I could refer to their property in the test name.

<img width="726" alt="Screenshot 2023-03-01 at 6 41 59 PM" src="https://user-images.githubusercontent.com/1937582/222235495-df051a21-a66c-4721-90df-858b2ec93d9f.png">